Pipfile

```diff
@@ -17,6 +21,8 @@ google-cloud-storage = "*"
 pandas = "*"
 matplotlib = "*"
 slackclient = "*"
+freezegun = "*"
+numpy = "*"
```

Comment:
These should be dev packages.

Reply:
Moved freezegun, removed numpy as I think I added it in error in the first place, it is already installed with pandas and not needed for the tests 👍